InstantReview/ThankYouPage.xaml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ContentPage xmlns="http://xamarin.com/schemas/2014/forms" xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml" x:Class="InstantReview.ThankYouPage">
+    <ContentPage.Content>
+        <RelativeLayout>
```

Comment:
Is a relative layout necessary here? Since we're just displaying a short set of strings on a page.

Reply:
You can remove them if you want.